packages/discord.js/src/structures/ActionRow.js

```diff
@@ -4,8 +4,11 @@ const { ActionRow: BuildersActionRow } = require('@discordjs/builders');
 const Transformers = require('../util/Transformers');
 
 class ActionRow extends BuildersActionRow {
-  constructor(data) {
-    super(Transformers.toSnakeCase(data));
+  constructor({ components, ...data } = {}) {
+    super({
+      components,
```

Comment:
This breaks camel cased components `new ActionRow({ components: [{type: 2, customId: '123', label: '123'}] })`.